docs/api.md

```diff
@@ -1707,6 +1708,7 @@ The method evaluates the XPath expression.
   - `url` <[string]> URL of a script to be added.
   - `path` <[string]> Path to the JavaScript file to be injected into frame. If `path` is a relative path, then it is resolved relative to [current working directory](https://nodejs.org/api/process.html#process_process_cwd).
   - `content` <[string]> Raw JavaScript content to be injected into frame.
+  - `type` <[string]> Script type. To be used with `url`. Use 'module' in order to load a Javascript ES6 module. See [script](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script) for more details.
```

Comment:
@yanivefraim shouldn't it also work with `path` and `content` options?

Reply:
@aslushnikov - you are right! added support for `path` and `content`. Thanks!